plugins/dependencies/src/main/kotlin/dependencies/DependencyNotationAndGroup.kt

```diff
@@ -1,9 +1,11 @@
 package dependencies
 
+import org.gradle.kotlin.dsl.IsNotADependency
+
 abstract class DependencyNotationAndGroup(
     group: String,
     name: String
-) : CharSequence {
+) : CharSequence, IsNotADependency {
```

Comment:
Actually, `DependencyNotationAndGroup` subclasses are a dependency, that's why they extend `CharSequence`.

Reply:
Ah ah! Now I get it